api/client/options.proto

```diff
@@ -102,8 +102,12 @@ message CommandLineOptions {
       [(validate.rules).uint32 = {gte: 1, lte: 1000000}];
   // The maximum allowed number of concurrent connections per event loop. HTTP/1 only. Default: 100.
   google.protobuf.UInt32Value connections = 2 [(validate.rules).uint32 = {gte: 1, lte: 1000000}];
-  // The number of seconds that the test should run. Default: 5. 0 will be treated as infinite.
-  google.protobuf.Duration duration = 3 [(validate.rules).duration.gte.nanos = 0];
+  oneof oneof_duration_options {
+    // The number of seconds that the test should run. Default: 5.
```

Comment:
Should this be a oneof in the proto, or we don't do / support that?

Reply:
I was looking at the right revision, but my eyes betrayed me. Nothing to see here, all good ;)